packages/amplify-codegen-appsync-model-plugin/src/visitors/appsync-typescript-visitor.ts

```diff
@@ -216,7 +216,11 @@ export class AppSyncModelTypeScriptVisitor<
   }
 
   protected getListType(typeStr: string, field: CodeGenField): string {
-    return `${typeStr}[]`;
+    let type: string = typeStr;
+    if (field.isList && field.isNullable) {
```

Comment:
I'm not familiar with the codegen code so treat accordingly, but would:

```suggestion
    if (field.isNullable) {
```

Be sufficient here?

Do we need to check the `isList` type since this function is always returning an array type regardless (line 223) and so it seems this if-block only cares if the list element type is nullable or not.

Reply:
Yeah I imagine just isNullable is sufficient, I was originally placing this code elsewhere which actually needed it until I found getListType. I am also learning the library haha